week_2/Assignment/AtHome/Error.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+type customError struct {
```

Comment:
It's better if we specify the error to indicate what error is, examples:
- type ErrFileNotFound struct{...}
- type ErrFileAccess struct{...}